### PR TITLE
Exercise Two css grid

### PR DESCRIPTION
**Pull request by AndresR9707**

### CODE REVIEW

File: 4-css-grid/exercise.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta name="og:title" property="og:title" content="grid">
+        <meta name="robots" content="2">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        </head>
+<body>
+    <main class="container">
+            <div class="box header">
```
Comment (pushdev-code) [inline 4-css-grid/exercise.html:12]: use semantic tags.
```suggestion
            <header class="box header">
```

File: 4-css-grid/exercise.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                <p>Header</p>
+            </div>
+            <div class="box leftS">
```
Comment (pushdev-code) [inline 4-css-grid/exercise.html:15]:
```suggestion
            <aside class="box leftS">
```

File: 4-css-grid/exercise.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                <p>Left Sidebar</p>
+            </div>
+            <div class="box article">
+                <p>Article</p>
+            </div>
```
Comment (pushdev-code) [inline 4-css-grid/exercise.html:20]:
```suggestion
            <section class="box article">
                <article>Article</article>
            </section>
```

File: 4-css-grid/exercise.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            <div class="box rightS">
```
Comment (pushdev-code) [inline 4-css-grid/exercise.html:21]:
```suggestion
            <aside class="box rightS">
```

File: 4-css-grid/exercise.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+                <p>Right Sidebar</p>
+            </div>
+            <div class="box footer">
```
Comment (pushdev-code) [inline 4-css-grid/exercise.html:24]:
```suggestion
            <footer class="box footer">
```

File: 4-css-grid/styles.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
+.container {
+    display: grid;
+    grid-template-columns: 150px auto 150px;
+    grid-template-rows: repeat (3, 100px);
+    grid-gap: 1em;
```
Comment (pushdev-code) [inline 4-css-grid/styles.css:5]: Sort CSS properties from A-Z

**Review by pushdev-code — CHANGES_REQUESTED** [review]

File: 4-css-grid/exercise.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        <meta name="og:title" property="og:title" content="grid">
+        <meta name="robots" content="2">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        </head>
+<body>
+    <main class="container">
+            <header class="box header">
+                <p>Header</p>
+            </div>
```
Comment (fabianmarinog) [inline 4-css-grid/exercise.html:14]:
```suggestion
            </header>
```

File: 4-css-grid/exercise.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+            <aside class="box leftS">
+                <p>Left Sidebar</p>
+            </div>
```
Comment (fabianmarinog) [inline 4-css-grid/exercise.html:17]:
```suggestion
            </aside>
```

File: 4-css-grid/exercise.html [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+            <section class="box article">
+                <article>Article</article>
+            </section>
+            <aside class="box rightS">
+                <p>Right Sidebar</p>
+            </div>
```
Comment (fabianmarinog) [inline 4-css-grid/exercise.html:23]:
```suggestion
            </aside>
```

File: 4-css-grid/exercise.html [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+            <footer class="box footer">
+                <p>Footer</p>
+            </div>
```
Comment (fabianmarinog) [inline 4-css-grid/exercise.html:26]:
```suggestion
            </footer>
```

**Review by fabianmarinog — CHANGES_REQUESTED** [review]

**Pull request merged**